client/create.go

```diff
@@ -8,12 +8,22 @@ import (
 	"github.com/pinterest/knox"
 )
 
+func init() {
+	cmdCreate.Run = runCreate // break init cycle
+}
+
 var cmdCreate = &Command{
-	Run:       runCreate,
-	UsageLine: "create <key_identifier>",
+	UsageLine: "create [--key-template template_name] <key_identifier>",
 	Short:     "creates a new key",
 	Long: `
-Create will create a new key in knox with original data set as the primary data. Key data should be sent to stdin.
+Create will create a new key in knox with original data set as the primary data. There are two ways to provide data in order to create a new key.
+
+First way: key data is sent to stdin.
+Please use command "create <key_identifier>". 
+
+Second way: using supported tink key template to create a new tink keyset containing a single key, which will be used as the data directly.
+Please use command "create --key-template template_name <key_identifier>".
```

Comment:
for each of these commands, I'd prefix it with knox since that is how these commands will be triggered in practice. (i.e. `knox create --key-template template_name <key_id>` instead of  `create --key-template template_name <key_id>`) . 

This wording could be clearer. Something like "the key-template option can be used to specify a template to generate the initial primary key version, instead of stdin. For available key templates, run "knox key-templates".

Reply:
Thank you so much! I have revised my code according to your comment.